nextflow_schema.json

```diff
@@ -13,7 +13,6 @@
             "properties": {
                 "process_type": {
                     "type": "string",
-                    "default": "only_validate",
```

Comment:
Will this be an issue for the orca dag that triggers daily validation?

Reply:
Great catch, I'm not sure how that got deleted. 

The good thing is changes in the `nextflow_schema.json` will not affect the code execution in actuality (this param is initialized to `only_validate` in `main.nf`) but only the aesthetic appearance of what you see on Nextflow Tower. So deleting this default just makes the param box appear as a blank when you click `Launch` for a pipeline.